src/types/transactions.rs

```diff
@@ -1444,10 +1444,10 @@ pub mod construct {
     /// it.
     /// This is deliberately made private so that the inconsistent internal
     /// state does not leak.
-    struct TransactionBuilder {
-        header:  TransactionHeader,
-        payload: Payload,
-        encoded: EncodedPayload,
+    pub struct TransactionBuilder {
+        pub header:  TransactionHeader,
+        pub payload: Payload,
+        pub encoded: EncodedPayload,
```

Comment:
This is not a good change. This type is deliberately internal.

For construction without signing there is a module `construct` which has a "PreAccountTransaction". It is public, and has many construction methods.

Reply:
Ah I see these functions changed since the SDK version I started with.